components/atom/videoPlayer/src/index.js

```diff
@@ -17,7 +20,9 @@ const AtomVideoPlayer = forwardRef(({src = ''}, forwardedRef) => {
 AtomVideoPlayer.displayName = 'AtomVideoPlayer'
 
 AtomVideoPlayer.propTypes = {
+  controls: PropTypes.bool,
```

Comment:
shouldn't we stick to naming props in a boolean-ish way? ie `hasControls`? even if that's mapped down the component tree to some other prop / HTML tag...?